drama-queen/src/ui/components/orchestrator/tools/functions.tsx

```diff
@@ -101,16 +101,13 @@ export function getinitialSurveyUnit(
   }
 }
 
-export function getInitialData(
-  surveyUnit: SurveyUnit
-): LunaticData | undefined {
+export function getInitialData(surveyUnit: SurveyUnit): LunaticData {
   const initialData = surveyUnit.data
   if (!externalResourcesUrl) return initialData
   return {
-    COLLECTED: initialData?.COLLECTED,
-    CALCULATED: initialData?.CALCULATED,
+    ...initialData,
```

Comment:
`initialData` can be undefined ? If yes, you will have an error here I think. 

why not on line 105: 

```
const initialData = surveyUnit.data ?? {}
```

Reply:
We use two functions to init suveyUnit (and its data).
We simplify this to clarify the use case